internal/database/db.go

```diff
@@ -460,6 +466,62 @@ func (dbs *SQLdb) getArchived(user, filepath, checksum string) (string, int, err
 	return filePath, fileSize, nil
 }
 
+// GetUserFiles retrieves all the files a user submitted
+func (dbs *SQLdb) GetUserFiles(userID string) ([]*SubmissionFileInfo, error) {
+	var (
+		err   error = nil
+		count int   = 0
+	)
+
+	files := []*SubmissionFileInfo{}
+
+	for count == 0 || (err != nil && count < dbRetryTimes) {
+		files, err = dbs.getUserFiles(userID)
+		count++
+	}
+
+	return files, err
+}
+
+// getUserFiles is the actual function performing work for GetUserFiles
+func (dbs *SQLdb) getUserFiles(userID string) ([]*SubmissionFileInfo, error) {
+	dbs.checkAndReconnectIfNeeded()
+
+	files := []*SubmissionFileInfo{}
+	db := dbs.DB
+
+	const query = "SELECT inbox_path, status, created_at " +
+		"FROM local_ega.files " +
+		"WHERE elixir_id = $1;"
```

Comment:
```suggestion
	const query = "SELECT submission_file_path, status, created_at " +
		"FROM sda.files " +
		"WHERE submission_user = $1;"
```

Reply:
Right! Updated the query, although it's a bit more complicated than that, since the `sda.files` doesn't contain the status